source/hovers.js

```diff
@@ -0,0 +1,869 @@
+// import { stringify } from 'querystring';
+
+// import { NONAME } from 'dns';
+
+// import { execSync } from 'child_process';
+// import { cp } from 'fs';
+
+class Req {
+  // Variable which states if a file was added before.
+  file_added = 0;
+  // Variable which stores the elements on which 
+  // event listeners for images are put.
+  event_img = [];
+  // Variable which stores the elements on which 
+  // event listeners for double click are put.
+  event_dbl_click = [];
+  constructor() {}
+  // Function for cleaning all the cache (information added from the file) in case an error occurs.
+  static solve_bugs() {
```

Comment:
In this file the indentation looks like 2 spaces whereas in **view.js** file it looks like 4 spaces.
Please be consistent (this is why it is important to use a linter)!!!
Check how the rest of the project is written and use the correct indentation!